### ISSUE TITLE
Adding version clause into task-options! gives error

### ISSUE THREAD

**Issue by jgrzebyta** [degree9/boot-semver#64]
Affected version `1.6.0`.

I just wanted keep version and `:generate` encoded in `task-options!` clause but have error.

```
(task-options!
 version {:minor 'two :patch 'zero :generate 'rdf4j.version}
 pom {:project (get-env :project)}
 aot {:all true})
```

After `boot version` I have error:

```
Initial Project Version...: 0.2.0
Generating Version Namespace...: rdf4j.version
Current Build Version...: 0.2.0
                              java.lang.Thread.run              Thread.java:  748
java.util.concurrent.ThreadPoolExecutor$Worker.run  ThreadPoolExecutor.java:  617
 java.util.concurrent.ThreadPoolExecutor.runWorker  ThreadPoolExecutor.java: 1142
               java.util.concurrent.FutureTask.run          FutureTask.java:  266
                                               ...                               
               clojure.core/binding-conveyor-fn/fn                 core.clj: 1938
                                 boot.core/boot/fn                 core.clj: 1029
                               boot.core/run-tasks                 core.clj: 1019
       degree9.boot-semver.impl/version-impl/fn/fn                 impl.clj:  100
         degree9.boot-semver.impl/eval986/fn/fn/fn                 impl.clj:   84
                                               ...                               
clojure.lang.ArityException: Wrong number of args (3) passed to: impl/eval986/fn--987/fn--992/fn--993/spit-ns--994
 clojure.lang.ExceptionInfo: Wrong number of args (3) passed to: impl/eval986/fn--987/fn--992/fn--993/spit-ns--994
    file: "/tmp/boot.user1599353312149023011.clj"
    line: 35
```

**Comment by burn2delete** [comment]
Please try version `1.7.0-SNAPSHOT`

**Comment by jgrzebyta** [comment]
Fixed in `1.7.0-SNAPSHOT`
